test-apps/appui-test-app/standalone/src/frontend/index.tsx

```diff
@@ -380,6 +381,25 @@ export class SampleAppIModelApp {
             section: StatusBarSection.Right,
             content: <AppLanguageSelect />,
           }),
+          StatusBarItemUtilities.createCustomItem({
+            id: "booster.SnapMode2",
+            section: StatusBarSection.Center,
+            content: (
+              <SnapModeField
+                snapMode={SnapMode.NearestKeypoint}
+                snapModes={
+                  Object.values(SnapMode).filter(
+                    (snapMode) => snapMode !== SnapMode.Bisector
+                  ) as SnapMode[]
```

Comment:
This type-cast is not correct since the filtered array will include enum keys as strings https://www.typescriptlang.org/docs/handbook/enums.html#reverse-mappings